src/fake_rtc.c

```diff
@@ -82,3 +86,19 @@ u32 FakeRtc_GetSecondsRatio(void)
                                                   1;
 }
 
+STATIC_ASSERT((OW_FLAG_PAUSE_TIME == 0 || OW_USE_FAKE_RTC == TRUE), FakeRtcMustBeTrueToPauseTime);
+
+void PauseFakeRtc(void)
```

Comment:
I believe we prefix those with `Script`

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/4954/commits/57f8663f5faae5ae011cef57f3246d5907812991

Updated patch file in testing section but did not re-test